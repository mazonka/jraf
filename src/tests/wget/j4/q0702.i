command=jw md 0 /.jraf.sys/users : login a@b.c * : login admin@server.com * : logout 0 : logout 73d24a440544ac5e : read 0 /.jraf.sys/login