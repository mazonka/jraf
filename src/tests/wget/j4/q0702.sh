#!/bin/sh

cwd=`sh execdir.sh`
wr=$cwd/wroot
echo ""
echo "=="
find $wr/.jraf.sys/ -maxdepth 2 -type d | xargs -n1 ls -1
echo "=="
ls -1 $wr/.jraf.sys/login/
rm -rf $wr



