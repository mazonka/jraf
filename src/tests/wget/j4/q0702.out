OK /.jraf.sys/users : OK http://127.0.0.1:16000/jraf? : OK http://127.0.0.1:16000/jraf? : OK : OK : JRAF_FAIL system path
==
login
users
5aeb1f6c5cd9c44a
f82d3bb17612458c
==
5aeb1f6c5cd9c44a
f82d3bb17612458c
